packages/replay-next/components/inspector/values/ObjectRenderer.tsx

```diff
@@ -15,22 +13,11 @@ const MAX_PROPERTIES_TO_PREVIEW = 5;
 export default function ObjectRenderer({ context, object, pauseId }: ObjectPreviewRendererProps) {
   const { className, preview } = object;
 
-  const [properties, propertiesWereTruncated] = useMemo(() => {
-    if (preview == null) {
-      return [[], false];
-    }
-
-    return mergePropertiesAndGetterValues(
-      preview.properties || [],
-      preview.getterValues || [],
-      MAX_PROPERTIES_TO_PREVIEW
-    );
-  }, [preview]);
-
-  const showOverflowMarker = object.preview?.overflow || propertiesWereTruncated;
-
+  const properties = preview?.properties?.slice(0, MAX_PROPERTIES_TO_PREVIEW);
   let propertiesList: ReactNode[] | null = null;
-  if (context !== "nested") {
+  if (context !== "nested" && properties) {
```

Comment:
Previously we were merging those two lists. Now that we aren't, if `properties` is empty but `getterValues` is not, this conditional will fail and we'll render "{...}" instead of "{ <list> }"

That's why [the `object_preview-01` test](https://tests.replay.io/recording/objectpreview-01-expressions-in-the-console-after-time-warping--e78b3650-ef3f-42c4-a112-85b6088095bc?e2e=&apiKey=&commentId=&focusWindow=eyJiZWdpbiI6eyJwb2ludCI6Ijg0Mzc0ODIzOTU3OTc3OTQ0MjE4NTEwMDU5OTkxMjAzODQiLCJ0aW1lIjozMzEzfSwiZW5kIjp7InBvaW50IjoiNzM5OTAyMzAyMzY5OTIxOTYyMjczMTc2MzMwMjE2NDIzMDciLCJ0aW1lIjo2NjUwNX19&point=13954297809004117444498180966188422&primaryPanel=cypress&secondaryPanel=console&time=5502&viewMode=non-dev) is failing. What previously rendered as `Proxy{}` is now rendering as `Proxy{…}`

Maybe this change is "expected" and the test would just need to be updated. I'm not sure.

Reply:
That wasn't intentional, I'll fix it.